docs/Dockerfile

```diff
@@ -16,12 +16,20 @@ FROM ruby:2.4
 
 RUN apt-get update && apt-get install -y pandoc
 
+ENV UID 1000
```

Comment:
And where would $USER come from. It is not defined. 

So you are saying even after once cleaning the _build_ directory, it still does not work?